src/scripts/main.js

```diff
@@ -2,7 +2,25 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+for (let i = 0; i < people.length; i++) {
+  const newRow = document.createElement('tr');
+  let gender = '';
+
+  if (people[i].sex === 'm') {
```

Comment:
you can use ternary operator instead of if..else construction. this will shorten your code